tools/shared/src/main/scala/org/scalajs/core/tools/linker/analyzer/Infos.scala

```diff
@@ -487,6 +488,9 @@ object Infos {
             case CreateJSClass(cls, _) =>
               builder.addInstantiatedClass(cls.className)
 
+            case Transient(CallHelper(_, args)) =>
+              args.foreach(traverse)
+
```

Comment:
Waaa... This is indeed awful :P Consider adding a comment that this is should only happen in the refinement phase.